egs2/vctk/asr1/local/data_prep.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Tomoki Hayashi
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+# shellcheck disable=SC1091
+. ./path.sh || exit 1;
+
+num_dev=5
```

Comment:
For tts, it might be ok, but is it also good for asr as well? Given that 5 utterances can hardly cover various condition.

Reply:
pls see comments below.